src/main/java/com/image/UserImages.java

```diff
@@ -0,0 +1,36 @@
+package com.image;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class UserImages implements Images {
+
+    Images list;
+    User user;
+
+    public UserImages(Images list, User user) {
```

Comment:
@PavelNejinskiy How about `ImagesOwnedBy` and `owner` as an argument instead of a `user`?